### PR TITLE
Servlet Step 1 Done 

### PR DESCRIPTION
**Pull request by SofiaPSU**
I accidentally merged the work I did for step 1 onto main under the Sofia_Branch, so I created a new branch for week 2. Now I am trying to figure out how to undo that merge.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/NewServlet.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 
-@WebServlet("/servlet")
+@WebServlet("/step_one_servlet")
```
Comment (claudiapao10) [inline portfolio/src/main/java/com/google/sps/servlets/NewServlet.java:11]: It is more descriptive, looks great!

**Comment by GoogleyGabriel** [comment]
> I accidentally merged the work I did for step 1 onto main under the Sofia_Branch, so I created a new branch for week 2. Now I am trying to figure out how to undo that merge.

Thats no biggie -- dw about unmerging.

**Review by GoogleyGabriel — CHANGES_REQUESTED** [review]
Make this PR only about changing the .xml files, note that in the description, and remove this file. Make this file be part of another PR where it is actually being used.

**Review by claudiapao10 — APPROVED** [review]

**Comment by SofiaPSU** [comment]
Removed the pom.xml files from this pull request

**Review by GoogleyGabriel — APPROVED** [review]

**Comment by GoogleyGabriel** [comment]
Good job! Seems like you missed just one file -- but its not an issue. Keep up the good work

**Pull request merged**